docs/tensorboard_projector_plugin.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p>It might be worth including a sentence on how you identified the more closely associated words (search for a particular word, the neighbors are highlighted. These are words that are close to it in the embedding space).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3423/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>